Add CI workflows for car app library fundamentals

cartland opened the pull request:
This commit adds two new CI workflows to automate the build process for the 'Car App Library Fundamentals' codelab. These workflows are triggered manually with worflow_dispatch and on push and pull request events to the `main` branch.

The workflows perform the following steps:

- Checkout the code
- Set up JDK 17
- Set up Gradle
- Make gradlew executable
- Build the automotive app or the regular app

cartland committed:
Add CI workflows for car app library fundamentals

This commit adds two new CI workflows to automate the build
process for the 'Car App Library Fundamentals' codelab. These
workflows are triggered manually with worflow_dispatch and
on push and pull request events to the `main` branch.

The workflows perform the following steps:

- Checkout the code
- Set up JDK 17
- Set up Gradle
- Make gradlew executable
- Build the automotive app or the regular app

pflammertsma commented:
Do we need a workflow for [`build-a-parked-app`](https://github.com/android/car-codelabs/tree/main/build-a-parked-app) as well?

cartland committed:
Add CI workflows for parked app codelab

This commit adds two new CI workflows to automate the build
process for the 'Build and test a parked app for Android
Automotive OS' codelab. These workflows are triggered manually
with workflow_dispatch and on push and pull request events to the
`main` branch.

The workflows perform the following steps:

- Checkout the code
- Set up JDK 17
- Set up Gradle
- Make gradlew executable
- Build the automotive app or the regular app

cartland commented:
@pflammertsma I added build scripts for `build-a-parked-app` and fixed the working-directory for the others.
